src/Util/TableFinder.php

```diff
@@ -71,7 +82,7 @@ protected function getTablesToBake(CollectionInterface $collection, array $optio
 
                     $config = (array)ConnectionManager::getConfig($this->connection);
                     $key = isset($config['schema']) ? 'schema' : 'database';
-                    if ($config[$key] === $split[1]) {
+                    if (isset($split[0]) && $config[$key] === $split[1]) {
```

Comment:
Shouldn't this check $split[1] not 0?

Reply:
These were mostly to appease phpstan/psalm that were complaining about unchecked index access within the if block. I can add more isset() checks for 1 as well.